boxwise_flask/resolvers.py

```diff
@@ -55,4 +57,12 @@ def resolve_user(_, info, email):
     return response
 
 
-schema = make_executable_schema(type_defs, query, snake_case_fallback_resolvers)
+@mutation.field("createBox")
+def create_box(_, info, input):
```

Comment:
wasn't sure what input was at first. Maybe worth renaming to box_creation_input. Something along those lines.